src/libraries/System.Transactions.Local/tests/TransactionManagerTest.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+
+namespace System.Transactions.Tests
+{
+    public class TransactionManagerTest
+    {
+        [Fact]
+        public void DefaultTimeout_MaxTimeout_Set_Get()
+        {
+            TimeSpan tsDefault = TimeSpan.Parse("00:02:00");
+            TimeSpan tsMax = TimeSpan.Parse("00:30:00");
+
+            TransactionManager.DefaultTimeout = tsDefault;
+            Assert.Equal(tsDefault, TransactionManager.DefaultTimeout);
```

Comment:
In the pipeline run, test sometimes fails (expected 00:02:00, actual 00:01:00) at this line on mono platform. I think the error reproduces more than 80% of the time, I initially thought running the cases in parallel caused the problem, but after writing all the settings in one method, the problem persists. I have no idea how this happened, @StephenMolloy, @mconnew, any thoughts?